src/docs/sphinx/tutorials/step12/Tutorial.rst

```diff
@@ -0,0 +1,357 @@
+| :math:`C`        | Cohesion                | [MPa]            | 1.0                |
++------------------+-------------------------+------------------+--------------------+
+| :math:`\phi_i`   | Initial Friction Angle  | [degree]         | 15.27              |
++------------------+-------------------------+------------------+--------------------+
+| :math:`\phi_r`   | Residual Friction Angle | [degree]         | 23.05              |
++------------------+-------------------------+------------------+--------------------+
+| :math:`c_h`      | Hardening Rate          | [-]              | 0.01               |
++------------------+-------------------------+------------------+--------------------+
+| :math:`\Sigma_h` | Min Horizontal Stress   | [MPa]            | -9.0               |
++------------------+-------------------------+------------------+--------------------+
+| :math:`\Sigma_H` | Max Horizontal Stress   | [MPa]            | -11.0              |
++------------------+-------------------------+------------------+--------------------+
+| :math:`\Sigma_v` | Vertical Stress         | [MPa]            | -12.0              |
++------------------+-------------------------+------------------+--------------------+
+| :math:`a_0`      | Initial Well Radius     | [m]              | 0.1                |
++------------------+-------------------------+------------------+--------------------+
+| :math:`P_w`      | Traction at Well        | [MPa]            | -10.0              |
++------------------+-------------------------+------------------+--------------------+
+| :math:`P_wf`     | Fluid Pressure at Well  | [MPa]            | 10.0               |
++------------------+-------------------------+------------------+--------------------+
+| :math:`\rho_f`   | Fluid Density           | [kg/m\ :sup:`3`] | 1000.0             |
++------------------+-------------------------+------------------+--------------------+
+| :math:`\mu`      | Fluid Viscosity         | [Pa s]           | 0.001              |
++------------------+-------------------------+------------------+--------------------+
+| :math:`c_f`      | Fluid Compressibility   | [Pa\ :sup:`-1`]  | 2.09*10\ :sup:`-10`|
++------------------+-------------------------+------------------+--------------------+
+| :math:`\kappa`   | Matrix Permeability     | [m\ :sup:`2`]    | 1.0*10\ :sup:`-20` |
++------------------+-------------------------+------------------+--------------------+
+| :math:`\phi`     | Porosity                | [-]              | 0.3                |
++------------------+-------------------------+------------------+--------------------+
+
+
+Inspecting results
+---------------------------------
+
+As defined in the ``Events`` section, we run this simulation for 497640 seconds. In the above examples, we requested silo-format output files. We can therefore import these into VisIt and use python scripts to visualize the outcome. 
```

Comment:
Add a note that a non-dimensional time is used in the analytical solution, and the end time here leads to a non-dimensional end time of t* = 1.0 (or whatever).

Reply:
will add this